databricks/sdk/mixins/jobs.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+
+from databricks.sdk.service import jobs
+
+
+class JobsExt(jobs.JobsAPI):
+
+    def get_run(self,
+                run_id: int,
+                *,
+                include_history: Optional[bool] = None,
+                include_resolved_values: Optional[bool] = None,
+                page_token: Optional[str] = None) -> jobs.Run:
```

Comment:
Should we have a doccomment here explaining that this makes multiple requests to return all tasks/iterations when the underlying has more than 100?